pages/in/settings/index.vue

```diff
@@ -81,10 +81,10 @@
                     {{ $t('Register.form.walletAddress') }}:
                   </span>
                   <span
-                    v-if="getUserInfo.cosmosWallet"
+                    v-if="getUserInfo.likeWallet"
                     class="content"
                   >
-                    {{ getUserInfo.cosmosWallet }}
+                    {{ getUserInfo.likeWallet }}
```

Comment:
This is conflict with https://github.com/likecoin/like-co/pull/1620

Reply:
I have merged and rebase the feature branch, @WeiJun0827 please change accordingly thanks

Reply:
Rebased the branch, ready for review, thanks!